core/src/Cuda/Kokkos_Cuda_Instance.cpp

```diff
@@ -209,13 +209,18 @@ void CudaInternal::print_configuration(std::ostream &s) const {
   for (int i : get_visible_devices()) {
     cudaDeviceProp prop;
     KOKKOS_IMPL_CUDA_SAFE_CALL(cudaGetDeviceProperties(&prop, i));
-    s << "Kokkos::Cuda[ " << i << " ] " << prop.name << " capability "
-      << prop.major << "." << prop.minor
-      << ", Total Global Memory: " << human_memory_size(prop.totalGlobalMem)
-      << ", Shared Memory per Block: "
-      << human_memory_size(prop.sharedMemPerBlock);
+    s << "Kokkos::Cuda[ " << i << " ] " << prop.name;
     if (m_cudaDev == i) s << " : Selected";
-    s << '\n';
+    s << '\n'
+      << "  Capability: " << prop.major << "." << prop.minor << '\n'
+      << "  Total Global Memory: " << human_memory_size(prop.totalGlobalMem)
+      << '\n'
+      << "  Shared Memory per Block: "
+      << human_memory_size(prop.sharedMemPerBlock) << '\n'
+      << "  Can access system allocated memory: " << prop.pageableMemoryAccess
+      << '\n'
+      << "  Via Address Translation Service: "
+      << prop.pageableMemoryAccessUsesHostPageTables << '\n';
```

Comment:
Maybe,
```suggestion
      << "    via Address Translation Service: "
      << prop.pageableMemoryAccessUsesHostPageTables << '\n';
```
to make clear that this belongs to the previous entry.